apps/angular/8-pure-pipe/src/app/heavy-computaion.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'heavyComputation',
+  standalone: true,
+})
+export class HeavyComputation implements PipeTransform {
+  transform(value: string, index: number): string {
+    return `${value} - ${index}`;
```

Comment:
Just to be on the safer side - can we add a null check here? Everything else is great!

Something like: 
````
transform(name: string | null | undefined, index: number | null | undefined): string {
    if (name == null || index == null) {
      return '';
    }
    
    return `${name} - ${index}`;
  }
  ````

Reply:
Thanks for the review @srikarphanikumar.